apiclient/__init__.py

```diff
@@ -10,7 +10,11 @@
 from googleapiclient import http
 from googleapiclient import mimeparse
 from googleapiclient import model
-from googleapiclient import sample_tools
+try:
+    from googleapiclient import sample_tools
+except ImportError:
+    import sys
+    sys.stderr.write("""Warning: unable to import apiclient.sample_tools\n""")
```

Comment:
Don't write to stderr, just ignore it. It'll give a standard import error if someone tries to `from apiclient import sample_tools`, and I'm with that (as they should be using `from googleapiclient import sample_tools`.